run_tests/README.md

```diff
@@ -0,0 +1,49 @@
+# Automated Testing Script
+
+This folder contains a script that will run through the tests in this repo using an automated UI integration testing approach. Because this uses a real browser with real hardware, it must be run on a machine with bluetooth functionality and a bluetooth device within range.
+
+At this time, only the Chrome implementation of this script is complete. There are placeholders for future Firefox and IE implementations.
+
```

Comment:
Explain somewhere how the puppeteer APIs for controlling the chooser are used. Cite their reference pages (is that https://pptr.dev/api/puppeteer.page.waitfordeviceprompt ?) and explain usage some. 

Goal is that a web developer who wants to automate testing of a Bluetooth app can see this example almost as a tutorial for how to do it.

Reply:
Hi @scheib I wrote a draft of this, could you please take a look and let me know if that looks like the right tone and level of detail?